src/pages/cli/graphql/data-modeling.mdx

```diff
@@ -500,7 +503,7 @@ The `@model` directive will generate:
 - Filter input objects that allow you to filter objects in list queries and relationship fields.
 - For list queries the default number of objects returned is 100. You can override this behavior by setting the limit argument.
 
-Type definition of the `@model` directive:
+### Type definition of the `@model` directive
```

Comment:
Suggestions: Elevate all pages' "How it works" into H2s and then make he definitions H3s